packages/fast-components-class-name-contracts-msft/src/carousel/index.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * The class name contract for the carousel component
+ */
+
+export interface CarouselClassNameContract {
+    /**
+     * The root of the carousel component
+     */
+    carousel?: string;
+
+    /**
+     * The collection of tabs and tab panels
+     */
+    carousel_slides?: string;
+
+    /**
+     * The collection of sequence indicators
+     */
+    carousel_sequenceIndicators?: string;
+
+    /**
+     * The sequence indicator
+     */
+    carousel_sequenceIndicator?: string;
+
+    /**
+     * The active carousel sequence indicator
+     */
+    carousel_sequenceIndicator__active?: string;
+
+    /**
+     * The collection of tab panels
+     */
+    carousel_tabPanels?: string;
+
+    /**
+     * The tab panel
+     */
+    carousel_tabPanel?: string;
+
+    /**
+     * The hidden tab panel
+     */
+    carousel_tabPanel__hidden?: string;
+
+    /**
+     * The tab panel content
+     */
+    carousel_tabPanelContent?: string;
+
+    /**
+     * The carousel's previous flipper
+     */
+    carousel_flipperPrevious?: string;
+
+    /**
+     * The carousel's next flipper
+     */
+    carousel_flipperNext?: string;
+
+    /**
+     * The light slide theme
+     */
+    theme__light?: string;
+
+    /**
+     * The dark slide theme
+     */
+    theme__dark?: string;
```

Comment:
Should these theme values be namespaced with carousel to follow our conventions?